src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,33 @@
+package qna.domain;
+
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * packageName : qna.domain
+ * fileName : Answers
+ * author : haedoang
+ * date : 2021/11/13
+ * description : 답변 일급 컬렉션
+ */
+@Embeddable
+public class Answers {
+    @OneToMany(mappedBy = "question")
+    private final List<Answer> answers = new ArrayList<>();
+
+    public void addAnswer(Answer answer) {
+        this.answers.add(answer);
+    }
+
+    public boolean hasAnswers() {
+        return this.answers.size() > 0;
+    }
+
+    public List<Answer> deletedFalseAnswers() {
```

Comment:
```suggestion
    public List<Answer> notDeletedAnswers() {
```

deletedFalse는 너무 데이터 관점의 네이밍으로 보여지는데, 상태를 나타내는 네이밍으로 바꿔보면 어떨까요?

Reply:
감사합니다!!